src/NavItem.js

```diff
@@ -28,10 +28,10 @@ class NavItem extends React.Component {
   }
 
   handleClick(e) {
-    if (this.props.onSelect) {
+    if (this.props.disabled) {
```

Comment:
You're right -- I'm not sure what should happen with ```disabled```? Or is ```preventDefault``` simply not needed at all? Feel free to correct the logic.